custom_components/localtuya/climate.py

```diff
@@ -335,14 +339,14 @@ def min_temp(self):
         """Return the minimum temperature."""
         if self.has_config(CONF_MIN_TEMP_DP):
             return self.dps_conf(CONF_MIN_TEMP_DP)
-        return DEFAULT_MIN_TEMP
+        return self._config[CONF_TEMP_MIN]
```

Comment:
Since this is optional, what if the user didn't set it? You need to check if it was configured, and return DEFAULT_MIN_TEMP otherwise.